packages/flutter/test/material/checkbox_test.dart

```diff
@@ -1587,6 +1587,7 @@ void main() {
 
   testWidgets('Checkbox has default error color when isError is set to true - M3', (WidgetTester tester) async {
     final FocusNode focusNode = FocusNode(debugLabel: 'Checkbox');
+    final ThemeData themeData = ThemeData(useMaterial3: true);
```

Comment:
Should use this themeData as the theme parameter in the MaterialApp below.

Reply:
Fixed! Thanks:)